ci/test.lua

```diff
@@ -379,7 +379,7 @@ local function load_tests(file, tests)
         dfhack.printerr('Skipping tests for unspecified target in ' .. file)
         return true  -- TODO: change to false once existing tests have targets specified
     end
-    local targets = type(env.config.targets) == table and env.config.targets or {env.config.targets}
+    local targets = type(env.config.target) == table and env.config.target or {env.config.target}
```

Comment:
`type()` returns strings:
```suggestion
    local targets = type(env.config.target) == 'table' and env.config.target or {env.config.target}
```

Reply:
ha whoops. thanks. fixed.